Recipes/TrainingDemo.Workflows.Sample.recipe.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "Lombiq.TrainingDemo.Workflows.Sample",
+  "displayName": "Training Demo - Sample Workflow - Person Page Content Item Created Logger",
+  "description": "A sample workflow that logs a message when a user creates a PersonPage Content Item.",
+  "author": "Lombiq Technologies",
+  "website": "https://github.com/Lombiq/Orchard-Training-Demo-Module",
+  "version": "1.0",
+  "issetuprecipe": false,
+  "categories": [
+    "training"
+  ],
+  "tags": [
+    "developer",
+    "training",
+    "sample"
+  ],
+  "steps": [
+    {
+      "name": "WorkflowType",
+      "data": [
+        {
+          "WorkflowTypeId": "4vtt0g09pbnt8x72gnc6557smk",
+          "Name": "Person Page Content Item Created Logger",
+          "IsEnabled": true,
+          "IsSingleton": false,
+          "LockTimeout": 0,
+          "LockExpiration": 0,
+          "DeleteFinishedWorkflows": false,
+          "Activities": [
+            {
+              "ActivityId": "4jgzsrtras0wt5byehn5s9j04h",
+              "Name": "LogTask",
```

Comment:
This should rather use the new custom activity (demonstrating that is the point), as under https://github.com/Lombiq/Orchard-Training-Demo-Module/pull/91#discussion_r995591221.